docs/guide/install.rst

```diff
@@ -23,8 +23,8 @@ Alternatively, you can use `pypa/pip`_, preferably in a `virtual environment`_:
 
    pip install jaxsim
 
-Have a look to `setup.cfg` for a complete list of optional dependencies.
-You can install all of them by specifying ``jaxsim[all]``.
+Have a look to ``pyroject.cfg`` for a complete list of optional dependencies.
```

Comment:
```suggestion
Have a look to ``pyproject.cfg`` for a complete list of optional dependencies.
```

Reply:
This will fail with:
```python
rst ``code`` is two backticks............................................Failed
- hook id: rst-backticks
- exit code: 1

docs/guide/install.rst:26:Have a look to `pyroject.cfg` for a complete list of optional dependencies.
```

What about:

```suggestion
Have a look to `pyroject.cfg`_ for a complete list of optional dependencies.
```

Reply:
I didn't notice it was a rst file. My suggestion wa mainly the name and the file extension.

Reply:
Sorry, I haven't noticed that either :)